plugin.py

```diff
@@ -112,10 +124,38 @@ def run() -> None:
 
         sublime.set_timeout_async(run)
 
-    def m_dart_textDocument_publishClosingLabels(self, params: Any) -> None:
-        # TODO: Implement me.
-        pass
+    def closing_labels(self, view: sublime.View, labels: List[Any]) -> Optional[List[sublime.Phantom]]:
+        phantoms = []
+        for label in labels:
+            final_character_position = label["range"]["end"]
+            if final_character_position["character"] < 1:
```

Comment:
maybe add a comment why we want to skip labels that have a character less than 1.

Maybe instead of `return` we can use `continue`
```
 if final_character_position["character"] < 1:
     continue
```
that way we can change the signature from 
```
-    def closing_labels(self, view: sublime.View, labels: List[Any]) -> Optional[List[sublime.Phantom]]:
+    def closing_labels(self, view: sublime.View, labels: List[Any]) -> List[sublime.Phantom]:
```

Reply:
this section has been taken from the [vscode's Dart extension](https://github.com/Dart-Code/Dart-Code/blob/57ead070d6a96d42839b6c6fe148716d9f8b97f1/src/extension/lsp/closing_labels_decorations.ts#L66), they chose to stop the loop instead of keeping looping through so I believe this is the *correct* way of handling such case  :smile: